package.json

```diff
@@ -71,7 +70,18 @@
     "cpx": "^1.5.0",
     "cross-env": "^7.0.0",
     "env-cmd": "^10.1.0",
-    "prettier": "^1.19.1",
+    "husky": "^4.3.0",
+    "lint-staged": "^10.5.0",
     "rimraf": "^3.0.2"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"

```

Comment:
Why can't we use `npm run lint` over here?

Reply:
Using `lint-staged` will only run eslint on files to be committed. Directly using `npm run lint` will run it on all files making it slow.

Reply:
Oh, that's good! Can you also do this in Meshery 😄 ?

Reply:
Okay